examples/extensions/update_sitelink_campaign_extension_setting.py

```diff
@@ -51,17 +51,18 @@ def main(client, customer_id, campaign_id, feed_item_ids):
         "ExtensionFeedItemService", version="v6"
     )
 
+    campaign_extension_setting = campaign_extension_setting_operation.update
     extension_feed_items = map(
         lambda feed_item_id: extension_feed_item_service.extension_feed_item_path(
             customer_id, feed_item_id
         ),
         feed_item_ids,
     )
-    campaign_extension_setting = campaign_extension_setting_operation.update
+    # Replace the current extension feed items with the given list
+    campaign_extension_setting.extension_feed_items[:] = extension_feed_items
```

Comment:
See my other comment on using slice v. extend - and I'll leave it up to you, but stylistically we would probably use a list comprehension here instead of map, so for example:

```py
campaign_extension_setting.extension_feed_items.extend([
    extension_feed_item_service.extension_feed_item_path(
            customer_id, feed_item_id
    ) for feed_item_id in feed_item_ids
])
```

Reply:
agreed this looks cleaner and is more concise